platform/mellanox/mlnx-platform-api/sonic_platform/device_data.py

```diff
@@ -153,7 +153,14 @@
         'sfp': {
             'max_port_per_line_card': 16
         }
-    }
+    },
+    'x86_64-nvidia_sn2201-r0': {
+        'thermal': {
+            "capability": {
+                "comex_amb": False
+            }
+        },
+    },
```

Comment:
not sure comma is needed here.

Reply:
Actually it is harmless but I removed it to keep the convention.